addons/l10n_in_ewaybill_stock/models/stock_move.py

```diff
@@ -17,31 +17,38 @@ class StockMove(models.Model):
     ewaybill_price_unit = fields.Monetary(
         compute='_compute_l10n_in_ewaybill_price_unit',
         currency_field='company_currency_id',
-        store=True
+        store=True,
+        readonly=False
     )
     ewaybill_tax_ids = fields.Many2many(
         comodel_name='account.tax',
         string="Taxes",
         compute='_compute_l10n_in_tax_ids',
-        store=True
+        store=True,
+        readonly=False
```

Comment:
I do not like users changing it manually on the ewaybill, because they risk doing a lot of manual work there, while it could come from the product.  The difference between inventory valuation and this might be too big, ...  It is not clearly analysed what should be the exact purpose of the prices on the ewaybills.  @tsb-odoo @ajjo-odoo

Reply:
> I do not like users changing it manually on the ewaybill, because they risk doing a lot of manual work there, while it could come from the product. The difference between inventory valuation and this might be too big, ... It is not clearly analysed what should be the exact purpose of the prices on the ewaybills. @tsb-odoo @ajjo-odoo

Hello @jco-odoo 

As per my understanding, 
We are about to store the value field over a eway bill model, and this will be exclusively used to send "Assessable Value" of goods to  E wayill bill portal.
It will not directly or indirectly effect our any of picking valuation or landed cost or price difference.

are we alligen in code accordingly right , @jva-odoo @hamo-odoo .